portal-api/api-tests/v1/gef/validation/eligibilityCriteria.api-test.js

```diff
@@ -0,0 +1,101 @@
+const {
+  getAnsweredItems,
+  isAutomaticCover,
+  eligibilityCriteriaStatus,
+} = require('../../../../src/v1/gef/controllers/validation/eligibilityCriteria');
+const { STATUS } = require('../../../../src/v1/gef/enums');
+
+describe('GEF controllers validation - eligibilityCriteria', () => {
+  describe('getAnsweredItems', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/UK-Export-Finance/dtfs2/pullRequest?prid=8286785)
```suggestion
  describe("getAnsweredItems", () => {
```